tests/Functional/tests/directive-title/directive-title.rst

```diff
@@ -1,3 +0,0 @@
-.. note::
-Some title!
-===========
```

Comment:
This is TOTALLY illegal (even indenting it is). This is the wrong behavior. I fixed it, and deleted the test (this test would now issue an error about the note not having any content, which is covered already in another test.